index.js

```diff
@@ -41,18 +44,6 @@ module.exports = function onload (el, on, off, caller) {
 module.exports.KEY_ATTR = KEY_ATTR
 module.exports.KEY_ID = KEY_ID
 
-function startObserving (obs) {
-  return function () {
-    obs.observe(document.body, {
```

Comment:
If I re-did the merge this morning, all I would do is change this to `document.documentElement`.

Reply:
Is all that needed though?

Reply:
I might be confused, let me re-read the OP.